### PR TITLE
Publish single systems page

### PR DESCRIPTION
**Pull request by frankharkins**
## Changes

Adds the single systems page to qiskit.org.

- [x] Description text
- [x] Prerequisites
- [x] Placeholder images for unfinished pages
- [x] https://github.com/Qiskit/qiskit.org/pull/2842
- [x] https://github.com/Qiskit/platypus/pull/1629

## Screenshots

Screenshot 2022-10-14 at 10 58 10

Screenshot 2022-10-14 at 10 58 31

### CODE REVIEW

File: components/learn/ExternalRecommendedReadings.vue [hunk 1]
```diff
@@ -21,8 +24,15 @@
...
-          {{ link.label }}
+          <span v-if="link.author">
+            {{ link.author }}
+          </span>
+          <span :class="link.author ? 'external-recommended-readings__title' : ''">
+            {{ link.label }}
+          </span>
         </AppLink>
+        <span v-if="link.description">-</span>
+        {{ link.description }}
```
Comment (techtolentino) [inline components/learn/ExternalRecommendedReadings.vue:35]: I think this should go within the `<span>` tags unless you saw an error that I might've missed.

**Review by techtolentino — CHANGES_REQUESTED** [review]
Very nice work here @frankharkins 

I've made a few comments/suggestions -- please let me know if you have any questions!

**Review by techtolentino — CHANGES_REQUESTED** [review]
@frankharkins

I also see these warnings in the terminal:

Screen Shot 2022-10-26 at 6 42 21 AM


To satisfy these errors, we'll need to add this to all the pages that extend CourseOverviewPage

`ExternalRecommendedReadingPreamble = ''`

For example, for this file: **quantum-simulation-summer-school-2022**

Screen Shot 2022-10-26 at 6 44 02 AM

File: components/learn/ExternalRecommendedReadings.vue [hunk 2]
```diff
@@ -3,6 +3,9 @@
     <h2>
```
[the remark this answers is not in the document]
Comment (frankharkins) [reply components/learn/ExternalRecommendedReadings.vue:3]: Awesome, this looks loads better, thanks

**Review by techtolentino — APPROVED** [review]
Awesome work here @frankharkins 

LGTM

🚢

**Pull request merged**